diff --git a/node_modules/hexo/lib/plugins/processor/post.js b/node_modules/hexo/lib/plugins/processor/post.js
index 01f274e..555d36d 100644
--- a/node_modules/hexo/lib/plugins/processor/post.js
+++ b/node_modules/hexo/lib/plugins/processor/post.js
@@ -165,7 +165,8 @@ module.exports = ctx => {
       const asset = PostAsset.findById(id);
 
       if (asset) return post.published === false ? asset.remove() : undefined; // delete if already exist
-      else if (post.published === false) return undefined; // skip assets for unpulished posts and
+      // TAG 草稿状态下的文章也生成资源目录
+      // else if (post.published === false) return undefined; // skip assets for unpulished posts and
 
       return PostAsset.save({
         _id: id,
